javatests/jsinterop/generator/externs/overloads/overloads.js

```diff
@@ -0,0 +1,41 @@
+
+/**
+ * @fileoverview Test "overloaded" method conventions
+ * @externs
+ */
+
+/**
+ * @constructor
+ */
+function Foo() {}
+
+Foo.noArgMethod = function() {};
+Foo.prototype.noArgMethod = function() {};
+
+/**
+ * @param {string|number} arg
+ */
+Foo.unionArgMethod = function(arg) {};
+/**
+ * @param {string|number} arg
+ */
+Foo.prototype.unionArgMethod = function(arg) {};
+
+/**
+ * @type {string|number}
+ */
+Foo.unionProperty;
+/**
+ * @type {string|number}
+ */
+Foo.prototype.unionProperty;
```

Comment:
I would put all test with union type into the uniontype test package.

Reply:
Will do, I wasn't sure if this justified its own test package or not.